components/docs-chef-io/content/habitat/environment_variables.md

```diff
@@ -15,7 +15,7 @@ This is a list of all environment variables that can be used to modify the opera
 
 | Variable | Context | Default | Description |
 |----------|----------|----------|-----------|
-| `HAB_AUTH_TOKEN` | build system | no default | Authorization token used to perform privileged operations against the depot, e.g. uploading packages or keys.
+| `HAB_AUTH_TOKEN` | build system | no default | Authorization token used to perform privileged operations against the depot, e.g. uploading packages or keys. Can also be configured in `~/.hab/etc/cli.toml` as `auth_token`.
```

Comment:
we don't really want people editing the cli.toml. It's just meant to be written to by `hab setup`.